diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewWrapper.kt b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewWrapper.kt
index 9f9d6f3..40913d7 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewWrapper.kt
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewWrapper.kt
@@ -17,6 +17,8 @@ class RNCWebViewWrapper(context: Context, webView: RNCWebView) : FrameLayout(con
     // We make the WebView as transparent on top of the container,
     // and let React Native sets background color for the container.
     webView.setBackgroundColor(Color.TRANSPARENT)
+
+    webView.requestFocus()
     addView(webView)
   }
 
