diff --git a/node_modules/react-native-animated-spinkit/src/Wander.tsx b/node_modules/react-native-animated-spinkit/src/Wander.tsx
index 7b9702a..04ab5ae 100644
--- a/node_modules/react-native-animated-spinkit/src/Wander.tsx
+++ b/node_modules/react-native-animated-spinkit/src/Wander.tsx
@@ -1,27 +1,22 @@
-import * as React from 'react'
-import { Animated, View } from 'react-native'
-import { SpinnerProps, defaultProps } from './SpinnerProps'
-import AnimationContainer from './AnimationContainer'
-import { loop } from './utils'
+import * as React from 'react';
+import {Animated, View} from 'react-native';
+import {SpinnerProps, defaultProps} from './SpinnerProps';
+import AnimationContainer from './AnimationContainer';
+import {loop} from './utils';
 
 export default class Wander extends React.Component<SpinnerProps> {
-  static defaultProps = defaultProps
+  static defaultProps = defaultProps;
 
   render() {
-    const {
-      size,
-      color,
-      style,
-      animating,
-      hidesWhenStopped,
-      ...rest
-    } = this.props
-    const wanderDistance = size * 0.75
+    const {size, style, animating, hidesWhenStopped, ...rest} = this.props;
+    const wanderDistance = size * 0.8;
+
+    const colors = ['#FFB900', '#E81123', '#0078D7'];
 
     return (
       <AnimationContainer
         initAnimation={() => ({
-          wander: (value) => ({
+          wander: value => ({
             values: [value],
             animation: loop({
               duration: 2000,
@@ -30,11 +25,11 @@ export default class Wander extends React.Component<SpinnerProps> {
             }),
           }),
         })}
-        animating={animating}
-      >
-        {(values) => (
+        animating={animating}>
+        {values => (
           <View
             style={[
+              // eslint-disable-next-line react-native/no-inline-styles
               {
                 width: size,
                 height: size,
@@ -42,13 +37,13 @@ export default class Wander extends React.Component<SpinnerProps> {
               },
               style,
             ]}
-            {...rest}
-          >
+            {...rest}>
             {Array(3)
               .fill(null)
               .map((_, index) => (
                 <View
                   key={index}
+                  // eslint-disable-next-line react-native/no-inline-styles
                   style={{
                     width: size,
                     height: size,
@@ -58,13 +53,13 @@ export default class Wander extends React.Component<SpinnerProps> {
                         rotate: `${index * 90}deg`,
                       },
                     ],
-                  }}
-                >
+                  }}>
                   <Animated.View
+                    // eslint-disable-next-line react-native/no-inline-styles
                     style={{
-                      width: size / 5,
-                      height: size / 5,
-                      backgroundColor: color,
+                      width: size / 4,
+                      height: size / 4,
+                      backgroundColor: colors[index],
                       position: 'absolute',
                       transform: [
                         {
@@ -120,6 +115,6 @@ export default class Wander extends React.Component<SpinnerProps> {
           </View>
         )}
       </AnimationContainer>
-    )
+    );
   }
 }
